### DISCUSSION TITLE
Welcome to the Keynotes Category for Universe 2022!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/events#16] [Keynotes]
🪐 Join GitHub CEO Thomas Dohmke for a special Universe 2022 kick-off live from the Yerba Buena main stage, where he and other GitHub leaders will discuss the future of software development (hint: the future is now.) We'll also explore how the best companies, developers, and organizations are using GitHub to transform industries and are setting the stage for a more optimistic future. For questions about this space check out the [FAQs](https://github.com/community/events/discussions/1) and don't forget you can [tune in live](http://watch.githubuniverse.com/). Let us know what sessions you're attending or loving below 🎤 !

**Comment by ovflowd** [comment]
I am super 🎉 excited for the Keynote. What will GitHub bring this year? 👀

**Reply by cryinkfly** [reply]
I'm also very excited about that. :+1:

**Reply by Sammorrison631** [reply]
I am so excited for this!!!

**Reply by vishalkumargeed** [reply]
> I am super 🎉 excited for the Keynote. What will GitHub bring this year? 👀

same here !

**Reply by AsfandSoomro** [reply]
Excited as well

**Reply by mrdiamonddirt** [reply]
me too 😊

**Reply by JuanGidoni** [reply]
me too 🎉

**Reply by urienix** [reply]
me too ✨

**Reply by hrthealer** [reply]
So happy being part of this vast community, I would like to thank all those that have contributed to this event to make it possible for all the World to attend.

**Reply by Taiwrash** [reply]
Getting interesting already. Feeling the pain of not been there in-person!

**Reply by Chromechef** [reply]
Im a junior web dev hopeful and I cant be more excited to be here. Github is such a fantastic tool and community builder. Awesome to see what they plan for the future.

**Reply by Jishnu-Epics** [reply]
Happy

**Reply by Rishabh787** [reply]
love from INDIA for every team members
By- RISHABH SINGH

**Reply by Alexxmfs** [reply]
Happy

**Comment by mrdiamonddirt** [comment]
first time at one of these events 😍

**Reply by mrdiamonddirt** [reply]
really excited and currently watching the Keynote preshow, who else is popping a cherry 🍒

**Reply by hrthealer** [reply]
This is my 2nd event and already love most of the content that is offered in the channels. I am new to this arena and have a great love for community and learning the GitHub repositories offered to the community. I mostly work with Jupyter Libs, Python and Anaconda. I just love all the welcoming of all the members. I feel like a LiL Guy @ T.M.K    
Love the So. Cal

**Reply by Rishabh787** [reply]
it was also my first event 
OSM **it's really wonderful**

**Comment by JuanGidoni** [comment]
I'm waiting for it to start, how anxious haha 🎉👍😍
First time at one of these events from github  😮

**Comment by RezaRitm** [comment]
It's first github universe that I was watching and so exciting for it!

**Comment by nikhilreddydev** [comment]
Hello, What are the most popular sessions?

**Reply by ossanna16** [reply]
Hi there @nikhilreddydev! [This](https://githubuniverse.com/events/detail/virtual-schedule/27f0940b-436f-41c2-8c8f-f5154718c9b3) has been mentioned as a popular session a few times.

**Comment by Moulick** [comment]
Why is the stream on youtube 12 seconds ahead of the stream on the universe website?

https://www.youtube.com/watch?v=wKZg0DScucw

**Reply by urienix** [reply]
I'm not sure, but maybe the render, transmission or video buffering method affects the streaming speed

**Reply by Moulick** [reply]
The gap is about 20 seconds now 🤣

**Comment by AtitBimali** [comment]
Let's gooo

**Comment by bu7maydAbdu** [comment]
happy github universe day everyone

**Comment by ssbarnea** [comment]
While I love Discussions, that is clearly not a medium for live social interaction, a Matrix channel would have more suited for that.

**Reply by angusjellis** [reply]
it's like trying to have a convo on a teams _channel_ rather than a teams _chat_

**Comment by Jim-Salmons** [comment]
👏🤓👍🥳 So excited to be attending #GitHubUniverse virtually. Especially interested in #Copilot-related announcements and demoes as relates to #Accessibility support for #DisabledDevelopers. For more see: https://bit.ly/Copilot4DisDev

**Comment by sarmedone** [comment]
Really excited to see what is to come :)

**Comment by jaskarandji** [comment]
Very excited to listen keynote . It's my first time attending #Github Universe , enjoying it , and  listening some good music.

**Reply by ossanna16** [reply]
Good to see you here @jaskarandji!

**Comment by jessikaraw** [comment]
BRBRBRBRBRBR <3

**Comment by petrosD93** [comment]
Will the sessions be available through VOD to watch the ones that might be in parallel with other?

**Reply by Moulick** [reply]
yeah, they'll be there on the Github universe website

**Reply by hrthealer** [reply]
This is my 1st G.H. Universe to attend, has been an amazing experience and will look forward to attending live next year.

**Comment by REFSZIN** [comment]
I am super 🎉 excited for the Keynote

**Comment by iMandarin** [comment]
94 Million Developers on GitHub.... Woahhh🔥💯💯

**Comment by Chromechef** [comment]
Hello All! Im a 20 year Culinary professional now for the past year studying front end web development to change my career and hopefully create web apps to help better the restaurant industry. Why? Because programming is amazing and it is so similar to cooking. Being my first time at an event like this and very new to programming. Would anyone care to spare some time on sharing with myself and others who may be in the same boat (limited experience) as I am some advice on how to join an open source project to contribute to this amazing community?  Many thanks!

**Reply by ossanna16** [reply]
Hi there @Chromechef and welcome! What an inspirational story!

**Reply by ossanna16** [reply]
Here are some great resources for OSS contribution:

* https://yourfirstpr.github.io/
* https://github.com/firstcontributions/first-contributions

**Reply by Chromechef** [reply]
@ossanna16 Thank you I truly appreciate this.

**Comment by advish10** [comment]
can we get recording of keynote?

**Reply by Bestra** [reply]
I think the youtube link will stay up after the livestream is done, https://www.youtube.com/watch?v=wKZg0DScucw

**Reply by ssbarnea** [reply]
$5000/900 developers equals to about $5.5 each... a late.

**Comment by ssbarnea** [comment]
I closely watching the news on Copilot, while hoping to see some news regarding addressing licensing issues related to suggested code, especially as a huge amount of training data is GPL...

**Comment by aerabi** [comment]
Super excited about "Hey GitHub". 🎉 Does anyone know where should I sign up for it? 🤔

**Reply by iMandarin** [reply]
Hey,

You can sign up using the Following link:

https://githubnext.com/projects/hey-github

**Comment by iMandarin** [comment]
I can see myself using Hey GitHub more than Hey Siri!

https://twitter.com/CryptoMandarin/status/1590402012302438400?s=20&t=gCN-rOrwUWqVmikyV5BCAQ

**Comment by iMandarin** [comment]
Hey Everyone,

You can sign up for 'Hey GitHub!' using the following  link.

https://githubnext.com/projects/hey-github

**Reply by ossanna16** [reply]
Thank you for posting this @iMandarin!

**Comment by rcmtcristian** [comment]
Each and every Keynote has been wonderful 🎉 excited for the rest of the Conf

**Comment by richardcarrigan** [comment]
Has anyone found more info on the GitHub Accelerator? It sounds awesome, but I can't find any links or other info on it.

**Reply by samus-aran** [reply]
Hi @richardcarrigan 👋 
You can find more information about [GitHub Accelerator here](https://accelerator.github.com/)!

**Reply by ossanna16** [reply]
Hi there @richardcarrigan, not sure if this is what you are looking for but [this is a summary](https://github.blog/2022-11-09-everything-new-from-github-universe-2022/) of everything that was announced during the Universe keynote.

**Reply by richardcarrigan** [reply]
> Hi @richardcarrigan 👋 
> You can find more information about [GitHub Accelerator here](https://accelerator.github.com/)! 

Thank you, @samus-aran!

**Comment by tpak** [comment]
Does anyone have the link to signup for the Issues/Projects beta from the keynote?

**Reply by queenofcorgis** [reply]
Are you referring to the private betas for Tasklists and Roadmap? Find the waitlist [here](https://github.com/features/issues/signup).

**Reply by tpak** [reply]
Thank you @queenofcorgis
